litmus-portal/frontend/src/components/Header/index.tsx

```diff
@@ -1,75 +1,106 @@
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import React, { useEffect, useState, useCallback } from 'react';
-import { Menu, MenuItem, Divider, Grid } from '@material-ui/core';
+import { Divider, Box } from '@material-ui/core';
 import { useLocation } from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import useStyles from './styles';
 import CustomBreadCrumbs from '../CustomBreadCrumbs';
-import NotificationsPopperButton from '../NotificationDropdown';
-import ProfileDropdown from '../ProfileDropdown';
+import NotificationsDropdown from './NotificationDropdown';
+import ProfileDropdownSection from './ProfileDropdownSection';
+import { UserData } from '../../models/user';
+import { RootState } from '../../redux/reducers';
+import { NotificationIds, Project, Message } from '../../models/header';
 
 const Header = () => {
```

Comment:
If you have a functional component with does not have a prop taken from any interface, you can write 
```
const Header:React.FC = () => {}
```